### PR TITLE
Add sequence params handling in HTTP v2

### PR DESCRIPTION
**Pull request by GuanLuo**

### CODE REVIEW

File: src/clients/python/experimental_api_v2/examples/simple_http_v2_sequence_sync_infer_client.py [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+    count = 1
+    for value in values:
+        # Create the tensor for INPUT
+        value_data = np.full(shape=[batch_size, 1],
+                             fill_value=value,
+                             dtype=np.int32)
+        inputs = []
+        inputs.append(httpclient.InferInput('INPUT', value_data.shape, "INT32"))
+        # Initialize the data
+        # FIXME, negative value in binary form can't be handled properly,
```
Comment (deadeyegoodwin) [inline src/clients/python/experimental_api_v2/examples/simple_http_v2_sequence_sync_infer_client.py:57]: Is this true?

Comment (GuanLuo) [reply src/clients/python/experimental_api_v2/examples/simple_http_v2_sequence_sync_infer_client.py:57]: Yes, I filled a ticket for it https://jirasw.nvidia.com/browse/DLIS-1225

**Comment by deadeyegoodwin** [comment]
@GuanLuo Manually rebase this and I will merge

**Comment by GuanLuo** [comment]
Rebased

**Pull request merged**